CODEOWNERS

```diff
@@ -2,8 +2,8 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
 # Default owners for all files:
-*               @nirs
+*               @nirs @aesteve-rh
 
 # Documentation owners
-docs/           @nirs @oVirt/ovirt-documentation
-README.md       @nirs @oVirt/ovirt-documentation
+docs/           @nirs @aesteve-rh @oVirt/ovirt-documentation
+README.md       @nirs @aesteve-rh @oVirt/ovirt-documentation
```

Comment:
Note for later patch - we can remove @oVirt/ovirt-documentation, this does not work with github codeowners file (see the error above), and we never got any contribution anyway.

Reply:
Ok, I will post another patch removing it. I noticed the error, but since it was there before I didn't act on it, I actually wanted to ask.